x/community/client/cli/tx.go

```diff
@@ -64,3 +71,140 @@ func getCmdFundCommunityPool() *cobra.Command {
+			proposal, err := utils.ParseCommunityPoolLendDepositProposal(clientCtx.Codec, args[0])
+			if err != nil {
+				return err
+			}
+
+			deposit, err := parseInitialDeposit(cmd)
+			if err != nil {
+				return err
+			}
+
+			from := clientCtx.GetFromAddress()
+			msg, err := govtypes.NewMsgSubmitProposal(&proposal, deposit, from)
+			if err != nil {
+				return err
+			}
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().String(flagDeposit, "", "Initial deposit for the proposal")
+
+	return cmd
+}
+
+// NewCmdSubmitCommunityPoolLendWithdrawProposal implements the command to submit a community-pool lend withdraw proposal
+func NewCmdSubmitCommunityPoolLendWithdrawProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "community-pool-lend-withdraw [proposal-file]",
+		Args:  cobra.ExactArgs(1),
+		Short: "Submit a community pool lend withdraw proposal",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Submit a community pool lend withdraw proposal along with an initial deposit.
+The proposal details must be supplied via a JSON file.
+Note that --deposit below is the initial proposal deposit submitted along with the proposal.
+Example:
+$ %s tx gov submit-proposal community-pool-lend-withdraw <path/to/proposal.json> --deposit 1000000000ukava --from=<key_or_address>
+Where proposal.json contains:
+{
+  "title": "Community Pool Withdrawal",
+  "description": "Withdraw some KAVA from community pool!",
+  "amount": [
+    {
+      "denom": "ukava",
+      "amount": "100000000000"
+    }
+  ]
+}
+`,
+				version.AppName,
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			// parse proposal
+			proposal, err := utils.ParseCommunityPoolLendWithdrawProposal(clientCtx.Codec, args[0])
+			if err != nil {
+				return err
+			}
+
+			deposit, err := parseInitialDeposit(cmd)
+			if err != nil {
+				return err
+			}
+			from := clientCtx.GetFromAddress()
+			msg, err := govtypes.NewMsgSubmitProposal(&proposal, deposit, from)
+			if err != nil {
+				return err
+			}
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().String(flagDeposit, "", "Initial deposit for the proposal")
+
+	return cmd
+}
+
+func parseInitialDeposit(cmd *cobra.Command) (sdk.Coins, error) {
+	// parse initial deposit
+	depositStr, err := cmd.Flags().GetString(flagDeposit)
+	if err != nil {
+		return nil, fmt.Errorf("no initial deposit found. did you set --deposit? %s", err)
+	}
+	deposit, err := sdk.ParseCoinsNormalized(depositStr)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse deposit: %s", err)
+	}
+	if !deposit.IsValid() || deposit.IsZero() {
+		return nil, fmt.Errorf("no initial deposit set, use --deposit flag")
+	}
```

Comment:
not sure if this is appropriate. the cmd requires that you must include a nonzero initial deposit.
can adjust this func accordingly if we want to allow submission with no initial deposit.